.github/workflows/ci.yml

```diff
@@ -1,6 +1,10 @@
 name: 🔮 CI
 
-on: [push]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Il faudra peut-être également rajouter `merge_group` (mais on devrait pouvoir le confirmer assez vite en essayant de merger cette PR :sweat_smile: )
cf https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#triggering-merge-group-checks-with-github-actions

Reply:
Bien vue, c'est quasi-sûr que ça soit nécessaire mais pour la science on va tester :D.